examples/semver.peggy

```diff
@@ -0,0 +1,60 @@
+/**
+ * SemVer.org v2
+ * https://semver.org/spec/v2.0.0.html
+ * For unit tests see: https://github.com/dselman/peggy-semver
+ */
+semver
+  = versionCore:versionCore
+    pre:('-' @preRelease)?
+    build:('+' @build)?
+  {
+    return { versionCore, pre, build };
+  }
+
+versionCore
+  = major:$numericIdentifier '.' minor:$numericIdentifier '.' patch:$numericIdentifier
+  {
+    return {
+      major: parseInt(major, 10),
```

Comment:
Refactor parseInt into `numericIdentifier`.

Reply:
Moving parseInt to `numericIdentifier` may be not so good, because that rule is also used in contexts where you do not want a number, but only a match result. I'm fine to leave parsing here.

Reply:
then an intermediate rule that does the parsing, I think.

Reply:
Regardless, the `$` should go in `numericIdentifier`.